gum/arch-arm/gumarmreader.c

```diff
@@ -64,11 +63,16 @@ gum_arm_reader_try_get_indirect_jump_target (gconstpointer address)
       op1->type == ARM_OP_REG && op1->reg == ARM_REG_R12 &&
       op2->type == ARM_OP_IMM)
   {
-    /* I couldn't really find the documentation of WHY this
-     * should be shifted by 12, but it seems to be how both
-     * objdump and IDA decode.
-     */
-    result += (op2->imm << 12);
+    if (insn->detail->arm.op_count == 4)
+    {
+      /* I couldn't really find the documentation of WHY this
```

Comment:
Sorry, I missed this: Comments, unless single-line, should have a blank opening line, like:

```C
/*
 * Foo
 * Bar
 */
```